modules/client-configuration/pages/registration-pubcloud-script.adoc

```diff
@@ -0,0 +1,293 @@
+[[clients.ssh.script]]
+= Example Script for Registering Clients with SSH
+
+You can use the ``mgr-ssh-push-init`` script to register both traditional and Salt clients with SSH, or SSH tunnelling.
+Before you start, ensure that the SSH key of the {productname} Server is listed in the authorized-keys on the client.
+
+
+
+.Example: Script for Registering Clients with SSH
+----
+#!/bin/bash
```

Comment:
I got the impression from the original post from Michael that this had been modified from the standard one. He says:

> The regular mgr-ssh-push-init script does stricthostchecking. I have attached a
> version where there is an extra option to disable this. Also here you
> have take care that the SSH key of the SUSE Manager Server is already
> present in the authorized-keys of the client. I have tested this method
> also at customers and it works.

Reply:
I made a "diff" between the script installed on our reference server and this in the PR.
100% identical.

@mbrookhuis we need your input here

Reply:
Checked with @mcalmer. The patch is present in head uyuni, 4.0 and 4.1.

Note: this will only work if the SUSE Manager public key is in the ssh configuration of the salt-minion. The key should be present in /root/.ssh/know_hosts.